python/pyspark/sql/catalog.py

```diff
@@ -956,7 +959,7 @@ def isCached(self, tableName: str) -> bool:
 
         Throw an analysis exception when the table does not exist.
 
-        >>> spark.catalog.isCached("not_existing_table")
+        >>> spark.catalog.isCached("not_existing_table")  # doctest: +SKIP
```

Comment:
Should enable this back too. I will fix them all once https://github.com/apache/spark/pull/39882 is merged.